### PR TITLE
[Chinese] Using MOSN with Istio: an alternative data plane.

### PR DESCRIPTION
**Pull request by wangfakang**
@rootsongjc Plz review it,thx.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2020/mosn-proxy/index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+- NET/IO 作为网络层，监测连接和数据包的到来，同时作为 listener filter 和 network filter 的挂载点;
+- Protocol 作为多协议引擎层，对数据包进行检测，并使用对应协议做 decode/encode 处理;
+- Stream 对 decode 的数据包做二次封装为 stream，作为 stream filter 的挂载点;
+- Proxy 作为 MOSN 的转发框架，对封装的 stream 做 proxy 处理;
+
+## 为什么要使用 MOSN ？
+
+蚂蚁集团在进行 Mesh 改造前，已经预料到作为下一代蚂蚁集团的基础架构，Mesh 化势必带来革命性的变革以及演进成本，我们有非常宏大的蓝图：准备将原有的网络和中间件方面的各种能力重新沉淀和打磨，打造成为未来新一代架构的底层平台，承载各种服务通讯的职责。
+
+这是一个需要多年时间打造，满足未来五年乃至十年需求的长期规划项目，合作共建团队跨业务、SRE、中间件、基础架构等部门。我们必须有一个具备灵活扩展、高性能、满足长期演进的网络代理转发平面。Nginx、Envoy 在网络代理领域有非常长期的能力积累和活跃的社区，我们也同时借鉴了 Nginx、Envoy 等其他优秀的开源网络代理， 同时在研发效率、灵活扩展等方面进行了加强，同时在整个 Mesh 改造涉及到非常多的部门和研发人员，必须考虑到跨团队合作的落地成本，所以我们基于 GoLang 自研了云原生场景下的新型网络代理 MOSN。对于 GoLang 的性能，我们前期也做了充分的调研和测试，满足蚂蚁集团业务对性能的要求。
```
Comment (SataQiu) [inline content/zh/blog/2020/mosn-proxy/index.md:38]:
```suggestion
这是一个需要多年时间打造，满足未来五年乃至十年需求的长期规划项目，合作共建团队跨业务、SRE、中间件、基础架构等部门。我们必须有一个具备灵活扩展、高性能、满足长期演进的网络代理转发平面。Nginx、Envoy 在网络代理领域有非常长期的能力积累和活跃的社区，我们也同时借鉴了 Nginx、Envoy 等其他优秀的开源网络代理，同时在研发效率、灵活扩展等方面进行了加强，同时整个 Mesh 改造涉及到非常多的部门和研发人员，必须考虑到跨团队合作的落地成本，所以我们基于 GoLang 自研了云原生场景下的新型网络代理 MOSN。对于 GoLang 的性能，我们前期也做了充分的调研和测试，满足蚂蚁集团业务对性能的要求。
```

File: content/zh/blog/2020/mosn-proxy/index.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+$ istioctl manifest apply  --set .values.global.proxy.image="mosnio/proxyv2:1.5.2-mosn"   --set meshConfig.defaultConfig.binaryPath="/usr/local/bin/mosn"
+{{< /text >}}
+
+检查 Istio 相关 pod 服务是否部署成功：
+
+{{< text bash >}}
+$ kubectl get svc -n istio-system
+{{< /text >}}
+
+如果服务状态 STATUS 为 Running，则表示 Istio 已经成功安装，后面就可以部署 Bookinfo 事例了。
```
Comment (SataQiu) [inline content/zh/blog/2020/mosn-proxy/index.md:105]:
```suggestion
如果服务状态 STATUS 为 Running，则表示 Istio 已经成功安装，后面就可以部署 Bookinfo 示例了。
```

File: content/zh/blog/2020/mosn-proxy/index.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+## Bookinfo 示例
+
+可以通过 [MOSN with Istio](https://katacoda.com/mosn/courses/istio/mosn-with-istio) 的教程来进行 Bookinfo 事例的演示操作，另外在该教程中您也可以找到更多关于使用 MOSN 和 Istio 的说明。
```
Comment (SataQiu) [inline content/zh/blog/2020/mosn-proxy/index.md:109]:
```suggestion
可以通过 [MOSN with Istio](https://katacoda.com/mosn/courses/istio/mosn-with-istio) 的教程来进行 Bookinfo 示例的演示操作，另外在该教程中您也可以找到更多关于使用 MOSN 和 Istio 的说明。
```

File: content/zh/blog/2020/mosn-proxy/index.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+## 展望
+
+接下来，MOSN 不仅会持续兼容适配新版本的 Istio 的功能，而且还将在以下几个方面进行发展：
+
+- 作为微服务运行时，面向 MOSN 的编程使使得务更轻、更小、更快;
```
Comment (SataQiu) [inline content/zh/blog/2020/mosn-proxy/index.md:115]: 语句不顺~

Comment (wangfakang) [reply content/zh/blog/2020/mosn-proxy/index.md]: Done, thx.

File: content/zh/blog/2020/mosn-proxy/index.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
+---
+title: "在 Istio 中使用 MOSN：一个替代数据平面"
```
Comment (rootsongjc) [inline content/zh/blog/2020/mosn-proxy/index.md:2]: “一个替代数据平面”这个说法感觉有点奇怪，替代谁？替代数据平面的不一定是数据平面。可以说“另一种数据平面”。

File: content/zh/blog/2020/mosn-proxy/index.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+subtitle: "用于边缘或服务网格的云原生代理"
+description: "Istio 的另一个 Sidecar 代理。"
+publishdate: 2020-07-29
+attribution: "Wang Fakang (mosn.io)"
```
Comment (rootsongjc) [inline content/zh/blog/2020/mosn-proxy/index.md:6]: 中文页面的作者写中文名吧。

**Pull request merged**